Makefile

```diff
@@ -35,3 +42,12 @@ refine: format license
 .PHONY: gitbook
 gitbook:
 	@gitbook build ./manuals ./docs
+
+.PHONY: build-local
+build-local:
+	CGO_ENABLED=0 go build -trimpath $(GOFLAGS) -o bin/nirvana -ldflags "-s -w \
+	-X $(VERSION_PKG).version=$(VERSION) \
+	-X $(VERSION_PKG).branch=$(BRANCH) \
+	-X $(VERSION_PKG).gitCommit=$(GITCOMMIT) \
+	-X $(VERSION_PKG).gitTreeState=$(GITTREESTATE) \
+	-X $(VERSION_PKG).buildDate=$(BUILDDATE)" ./cmd/nirvana
```

Comment:
Should we also change the go template? cc @iawia002

Reply:
> go template

Can you be more specific?